testing/regress/ecl/parquetSize.ecl

```diff
@@ -0,0 +1,40 @@
+/*##############################################################################
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+##############################################################################*/
+
+//class=parquet
+
+IMPORT Parquet;
+
+recordLayout := RECORD
+    UNSIGNED4 id;
+    STRING name;
+    REAL8 price;
+    STRING isactive;
+END;
+
+// Paths to the files
+singleFilePath := '/var/lib/HPCCSystems/mydropzone/single.parquet';
```

Comment:
I don't see these files included in the PR. The files should be: single.parquet,  multi_1_of_3.parquet, multi_2_of_3.parquet, multi_3_of_3.parquet